resources/sass/app.scss

```diff
@@ -1,3 +1,6 @@
+// Font-awesome
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css");
```

Comment:
https://stackoverflow.com/questions/52376720/how-to-make-font-awesome-5-work-with-webpack
import theo link hướng dẫn ở trên